frontend/src/lib/components/PayGateMini/PayGateMini.tsx

```diff
@@ -157,7 +157,7 @@ function PayGateContent({
         <div
             className={clsx(
                 className,
-                background && 'bg-side border border-border',
+                background && 'bg-bg-3000 border border-border',
```

Comment:
probably copypasta typo? `bg-bg-3000` instead of `bg-3000`, more places are like this.

Reply:
Funnily enough it is actually correct.

We autogenerate classes for all our colors so you can do things like `bg-primary` or `bg-secondary` to mean "make the background the primary / secondary color". In this case the color is called `bg-3000` so the generated color is `bg-bg-3000`. Other examples like `text-bg-3000` would also work and would set the text to the background color.

It does read a little oddly and is confusing but probably best left for consistency (or rename the colour once all other cleanup is done)